fdapm/newrelic_middleware.go

```diff
@@ -16,7 +17,7 @@ func NewRelicMiddleware(app newrelic.Application) fdmiddleware.Middleware {
 			var txn newrelic.Transaction
 			txn = newrelic.FromContext(req.Context())
 			if txn == nil {
-				txn = app.StartTransaction(req.URL.Path, w, req)
+				txn = app.StartTransaction(fmt.Sprintf("%s %s", req.Method, req.URL.Path), w, req)
```

Comment:
```suggestion
				txn = app.StartTransaction(fmt.Sprintf("%s [%s]", req.URL.Path, req.Method), w, req)
```
WDYT? in that way you added the information, easier to search and the chenge is softer